### PR TITLE
Add instructions for getting `make` to work on Windows

### LINKED ISSUE #633: Makefile: `runghc` instead of `runhaskell`
To get `make` to run on my Windows 10 machine, I had to change line 5 of GNUmakefile from
```
RUNHASKELL?=runhaskell
```
to
```
RUNHASKELL?=runghc
```
I'm not sure if this is related to Windows, or to new versions of GHC. I'm running GHC version 9.0.1.

Is `runhaskell` still working for those on Unix?

### PR DESCRIPTION
**Pull request by barrettj12**
Fixes #633 (kinda)

Please squash and merge into a single commit.

### CODE REVIEW

File: MAKEWINDOWS.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Instructions for getting `make` to work on Windows
+
+The GNU command `make` does not work on Windows out of the box. Here are the steps I took to get it to work on my Windows 10 machine:
```
Comment (mortberg) [inline MAKEWINDOWS.md:3]: Please don't use "I" and instead write something generic like: "Here are some possible steps that can make it to work on Windows 10"

File: INSTALL.md [hunk 2]
```diff
@@ -86,7 +86,9 @@ Once this works go to a suitable directory and run
 > make
 ```
 
-This should compile all of the agda/cubical files. To test that it
+This should compile all of the agda/cubical files. (If you're on Windows, see [here](MAKEWINDOWS.md) for instructions on using `make`).
```
Comment (mortberg) [inline INSTALL.md:89]: Please keep length of lines short (you can use `M-q` in emacs)

File: INSTALL.md [hunk 3]
```diff
@@ -86,7 +86,9 @@ Once this works go to a suitable directory and run
...
+
+To test that it
```
Comment (mortberg) [inline INSTALL.md:91]: Why is there a line break?

File: INSTALL.md [hunk 4]
```diff
@@ -167,7 +171,9 @@ Once this works go to a suitable directory and run
 > make
 ```
 
-This should compile all of the agda/cubical files. To test that it
+This should compile all of the agda/cubical files. (If you're on Windows, see [here](MAKEWINDOWS.md) for instructions on using `make`).
```
Comment (mortberg) [inline INSTALL.md:174]: Please add line break

File: INSTALL.md [hunk 5]
```diff
@@ -167,7 +171,9 @@ Once this works go to a suitable directory and run
...
+
+To test that it
```
Comment (mortberg) [inline INSTALL.md:176]: Weird line break here as well

File: INSTALL.md [hunk 6]
```diff
@@ -251,7 +257,9 @@ Once this works go to a suitable directory and run
 > make
 ```
 
-This should compile all of the agda/cubical files. To test that it
+This should compile all of the agda/cubical files. (If you're on Windows, see [here](MAKEWINDOWS.md) for instructions on using `make`).
```
Comment (mortberg) [inline INSTALL.md:260]: Long line

File: INSTALL.md [hunk 7]
```diff
@@ -251,7 +257,9 @@ Once this works go to a suitable directory and run
...
+
+To test that it
```
Comment (mortberg) [inline INSTALL.md:262]: And the line break again :-)

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -8,6 +8,8 @@ When preparing a PR here are some general guidelines:
   which will generate all required `Everything` files in
   `Cubical/README.agda` and then typecheck the latter file.
 
+  - If you're using Windows, [here](MAKEWINDOWS.md) are some instructions to get the `make` command working.
```
Comment (mortberg) [inline CONTRIBUTING.md:11]: Please don't indent this line and add some line breaks

**Review by mortberg — CHANGES_REQUESTED** [review]
Lots of minor stylistic comments. I don't have windows so cannot test, but I think it's good to have these instructions in case someone else has problems

**Comment by Saizan** [comment]
By the way, I don't think `INSTALL.md` should tell users to use `make` for `agda/cubical`, it's not necessary to "install" the library.

In that case if you install Agda through cabal or stack you won't have to run `make` at all.

File: MAKEWINDOWS.md (hunk 1 above)
Comment (barrettj12) [reply MAKEWINDOWS.md:3]: Well, the point was that this worked on my machine, but I can't guarantee it's gonna work on anyone else's. But okay, I will rewrite

**Comment by barrettj12** [comment]
> By the way, I don't think `INSTALL.md` should tell users to use `make` for `agda/cubical`, it's not necessary to "install" the library.
> 
> In that case if you install Agda through cabal or stack you won't have to run `make` at all.

True, it's just to test that the installation worked right? I was thinking you could factor this out into a separate section, since this part seems to be repeated verbatim for each installation method.

**Comment by Saizan** [comment]
Yeah, `make` only tests that the installed `agda` can typecheck all the files of the library.

Btw, I still think it's helpful to know how to get make to work on windows, but probably more for someone contributing so that they can run the tests.

**Comment by barrettj12** [comment]
@mortberg fixed. @Saizan had a good point that this is not necessary in the `INSTALL.md` file, so I've reverted this to the current version.

**Review by mortberg — APPROVED** [review]

**Comment by mortberg** [comment]
Looks good to me!

**Pull request merged**